Gems/Prefab/PrefabDependencyViewer/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# 
+# Copyright (c) Contributors to the Open 3D Engine Project
+# 
```

Comment:
Seems like its missing "For complete information" line in the copyright header?

Reply:
Resolved.